### ISSUE TITLE
Newly created test DB tarball won't be restored in CirceCI workflow

### ISSUE THREAD

**Issue by tianrunhe** [ParabolInc/parabol#4119]
## Issue - Bug
To reproduce:
1. Create a new test DB tarball following the steps in [createFixture.ts](https://github.com/ParabolInc/parabol/blob/master/packages/cypress/fixtures/createFixtures.ts)
2. Commit the change and push it to a CirceCI connected branch, say `development`
3. Observe the workflow failed in "Cypress Tests" step and split out error messages like:

```
ReQL error during 'create table': Unrecognized secondary index version, secondary index not created.
In file: /tmp/tmpYwWaCz/rethinkdb_dump_2020-08-17T17:37:39/test/RetroReflectionGroup.json
ReQL error during 'create table': Unrecognized secondary index version, secondary index not created.
In file: /tmp/tmpYwWaCz/rethinkdb_dump_2020-08-17T17:37:39/test/SuggestedAction.json
```

  Can refer to a previously failed CirceCI workflow [here](https://app.circleci.com/pipelines/github/ParabolInc/parabol/650/workflows/380e9fcb-5d82-47e1-bdca-ca5f00cd5c8e/jobs/9528)

4. I've tried a couple of combination of rethinkdb versions between the one used to generate the tarball (i.e., modifying [here](https://github.com/petecoop/rethinkdb-driver/blob/master/Dockerfile#L3)) and the one we used in Cypress Test (i.e., [here](https://github.com/ParabolInc/parabol/blob/aa41628eee3da64ade500d06d22be7e30250330a/.circleci/config.yml#L70)) and got no luck

### Acceptance Criteria
- [ ] a new test DB tarball is accepted during Cypress Test step in CirceCI
- [ ] Developer can use the new test DB to run integration tests locally

**Estimated effort:** 16 hrs ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))

**Comment by ackernaut** [comment]
Matt’s saying 16 hours for this one, under DX, will put it in the backlog for prioritization.

**Comment by tianrunhe** [comment]
This issue is resolved after 68363ba.
